packages/core/src/common/document-cache.ts

```diff
@@ -58,8 +58,15 @@ export default class DocumentCache {
     );
   }
 
+  private gtsAwareExtName(filename: string): string {
+    if (filename.endsWith('.gts.ts')) {
```

Comment:
two extensions? 
:upside_down_face:

Reply:
This is the extension that TypeScript tries if you import a .gts module since it has no intrinsic understanding of the `.gts` extension. We're intercepting it to map it to the actual file.

Reply:
This is worth a comment in the source because it will definitely look weird to future readers.